### ISSUE TITLE
In windows, nushell malfunction with filenames with spaces

### ISSUE THREAD

**Issue by nosly** [nushell/nushell#9492]
### Describe the bug

In windows, to delete files with size == 0KB under a directory, we use
```nushell
ls | where size == 0KB and type == file | each { rm $in.name --permanent }
```
If any suitable files are having spaces in their names, they are quietly ignored, have tried to use ', " or back tick to surround the filename with no luck. The only work around is to use windows built-in `del` in place of `rm`
```nushell
ls | where size == 0KB and type == file | each { cmd /C del $'"($in.name)"' }
```

However, if we want to use `mv` or `cp` in similar way, will always get "no matches found" error. If we use windows native `copy` or `xcopy` or `robocopy` instead of `cp`, no matter how many suitable files (with spaces in name), it will always copy same 5 files and cannot find another 2 files and leave the rest of the suitable files alone

### How to reproduce

1. Create 2 empty folders A and B
2. In folder A, generate 50 files of 0KB and file name with spaces
3. `cd A`, run:
```nushell
ls | where size == 0KB and type == file | each { rm $in.name --permanent }
```
4. `cd A`, run:
```nushell
ls | where size == 0KB and type == file | each { cp $in.name B}
```

### Expected behavior

I expected `rm`, `mv` and `cp` to work as intended for filenames with spaces in nu in windows

### Configuration

~> version | transpose key value | to md --pretty                   
| key                | value                                    |
| ------------------ | ---------------------------------------- |
| version            | 0.81.0                                   |
| branch             |                                          |
| commit_hash        | 5c81bcef1a3c1667468c81ef8c06cf56a0962133 |
| build_os           | windows-x86_64                           |
| build_target       | x86_64-pc-windows-msvc                   |
| rust_version       | rustc 1.68.2 (9eb3afe9e 2023-03-27)      |
| rust_channel       | 1.68.2-x86_64-pc-windows-msvc            |
| cargo_version      | cargo 1.68.2 (6feb7c9cf 2023-03-26)      |
| build_time         | 2023-06-06 22:55:37 +00:00               |
| build_rust_channel | release                                  |
| features           | default, sqlite, trash, which, zip       |
| installed_plugins  |                                          |

### Additional context

Similar issue was first reported here: [https://github.com/nushell/nushell/issues/5420](url)

**Comment by cptpiepmatz** [comment]
I could not reproduce any issues with filenames with spaces. Seems to be fixed.

**Comment by nosly** [comment]
Yes, as at Version: 0.106.1 (x86_64-pc-windows-msvc), it seems fixed, it could have been fixed earlier, not sure since which version. Thank you
